### DISCUSSION TITLE
Any ideas on how to adjust the size of chart?

### DISCUSSION THREAD

**Discussion by Ieejiajia** [Live-Charts/LiveCharts2#1189] [Q&A]
Recently I am using the livechart 2 with the blazor platform. I am using the CartesianChart to plot my chart, but the size of chart are too small for me.  Any ideas on how to adjust the size of chart?  because I cannot adjust the chart using the width and height as this two is no the property of CartesianChart .

**Comment by upswing1** [comment]
I am not sure if that helps you but in MAUI I do it like this:
xaml
<lvc:CartesianChart Grid.Row="8" Grid.Column="{Binding GridColumn_Chart32}" Grid.ColumnSpan="{Binding GridColumnSpan_Chart3}" 
                                HeightRequest = "{Binding ChartHeightRequest}" WidthRequest="{Binding WidthRequest_Chart32}" BackgroundColor="White"
                                Series="{Binding BarSeries3}"
                                IsVisible="{Binding IsVisible_Chart32}"
                                TappedCommand="{Binding TappedCommand_Chart32}" >
            </lvc:CartesianChart>  

in code behind do this:
 protected override void OnSizeAllocated(double width, double height) // using this instead of SizeChanged because it is more optimized
    {
        base.OnSizeAllocated(width, height);

        // Check if the size is valid (sometimes OnSizeAllocated is called with invalid values)
        if (width > 0 && height > 0)
        {
            _viewModel.UpdateChartSize(width, height);
        }
    }
 in view model  you adjust based on your needs l:
inside UpdateChartSize method 
ChartWidthRequest = windowWidth / 3.23;
            ChartHeightRequest = windowHeigth / 3.52;

**Comment by coder-johnz** [comment]
```
<div style="width: 100%; height: calc(100vh - 250px);">
    <CartesianChart
        Series="series">
    </CartesianChart>
</div>
```
